momentum/4/modules/ha-proxy-client.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "ha_proxy_client - HAProxy protocol client module"
+description: "The ha_proxy_client module is used to configure Momentum to use HAProxy's PROXY protocol version 2 for outbound connections (see https://github.com/haproxy/haproxy/blob/master/doc/proxy-protocol.txt).  This can be leveraged in cases where your sending IPs are deployed on a different machine than the Momentum.
+---
+
+When configured for a binding or binding group Momentum will connect to the given `ha_proxy_server` and prefix the SMTP session with a PROXY protocol version 2 header.  The `dst_addr` and `dst_port` will be filled in with the resolved MX, the `src_addr` will be filled in with the configured value of `ha_proxy_src_addr`.  It is the customers responsibility to configure a listener on `dst_addr:dst_port` that listens for PROXY protocol and routes based on the `src_addr`, but a simple example configuration for HAProxy would be:
+
+```
+# listen to proxy protocol and forward to backend
+frontend  main
+    bind *:5000 accept-proxy
+    mode tcp
+    default_backend     out
+
+# forward to the destination using the clientip
+# from proxy protocol as the source address 
+backend out
+    mode tcp
+    source 0.0.0.0 usesrc clientip
+    server foo 0.0.0.0
+``` 
+
+## IPV6 Support
+
+Recently, **IPV6** support was introduced to the module. IPV6 can be proxied standlone or via an IPV4 HAProxy listener. To utilize IPV6, the following configuration(s) from above can be modified. 
```

Comment:
I don't think "proxied standalone" really makes sense.  I think it might make more sense to say something like:

```
If you need to proxy IPV6 traffic then you need to specify an `ha_proxy_ipv6_src_addr` for the binding or binding_group.  If you need to proxy both IPV4 and IPV6 over the same binding or binding_group you must specify *both* `ha_proxy_src_addr` and `ha_proxy_ipv6_src_addr`.
```